### DISCUSSION TITLE
有关跨域和后端发现的问题

### DISCUSSION THREAD

**Discussion by SSSSSSSan** [xfgryujk/blivechat#243] [提问]
现在允许跨域好像是写死在代码里头，允许*.blive.chat，没在配置文件看到相应的配置项，看起来我还是得针对我自己的环境做修改
然后后端发现这个会影响到我自建服务器么，我有些东西依赖插件引擎，这个后端发现是在服务端做的发现吧？web端受影响么

**Comment by SSSSSSSan** [comment]
跨域的看到了 正则...

**Comment by xfgryujk** [comment]
不知道你的需求，RTFSC吧

**Reply by SSSSSSSan** [reply]
im trying
